scenarioo-validator/src/test/java/org/scenarioo/validator/ScenariooValidatorTest.java

```diff
@@ -0,0 +1,28 @@
+package org.scenarioo.validator;
+
+import org.junit.Assert;
+import org.junit.Ignore;
+import org.junit.Test;
+
+import java.io.File;
+
+public class ScenariooValidatorTest {
+
+    /**
+     * Our validator must successfully validate our own example docu.
+     */
+    @Test
+    @Ignore("This is ignored since this test produces derived files in scenario-docu-generatio-example which is under version control")
```

Comment:
hmm, yes, we should start to implement better tests in server project and for validator as well.

Idea: we could introduce some test samples that we can import using the importer and the validator (using the clean function we could always delete the derived files and of course we could not refer to the files that are checked in but use the resources in the build folder instead, where it does not matter if something is changed). Like this we could easily improve tests in validator and in server (also for testing backward compatibility issues).

I open an issue for this with a proposal: #453